.github/workflows/dev_on_schedule_tag_and_deploy_staging.yml

```
@@ -4,12 +4,12 @@ on:
   schedule:
     # Winter time (CET, UTC+1): from end of October to end of March
     # 10:30 in France = 09:30 UTC
-    - cron: '30 9 * 10-12 *'
-    - cron: '30 9 * 1-3 *'
+    - cron: '30 9 * 10-12 4'
+    - cron: '30 9 * 1-3 4'
```

J'aurais plutôt écrit : 
```suggestion
    - cron: '30 9 * 1-3,11-12 4'
```
une seule ligne, et octobre reste quand même majoritairement en été !